tests/tape/test_qnode.py

```diff
@@ -813,8 +799,6 @@ class TestIntegration:
 
     def test_correct_number_of_executions_autograd(self):
         """Test that number of executions are tracked in the autograd interface."""
-        qml.enable_tape()

```

Comment:
On line 767 above, in docstring for `test_multi_thread`, there's a mention of "tape mode". Perhaps just skip those words since it's default now?

Reply:
good catch, deleted!